### PR TITLE
open control yaml 

### PR DESCRIPTION
**Pull request by LindsayYoung**
Making progress on this for ATO renual

### CODE REVIEW

File: opencontrol.yaml [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+  - key: ""
+    text: "Disabling interactive accounts is be handled automatically by cloud.gov. In the RDS databases and FEC cms by administrators and superuser accounts can manually do this."
+-
+ standard_key: NIST-800-53
+    # ACCOUNT MANAGEMENT | INACTIVITY LOGOUT
+    #The organization requires that users log out when [Assignment: organization-defined time-period of expected inactivity or description of when to log out]. Related to: SC-23
+ control_key: AC-2 (5)
+ narrative:
+  - key: ""
+    text: "Cloud.gov has inactivity log out the CMS and RDS boxes do not."
```
Comment (ccostino) [inline opencontrol.yaml]: Should `out` be `of`?

File: opencontrol.yaml [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+       - key: ""
+         text: "Cloud.gov logs are available for audit. New Relic also has error reporting notifications that are sent to developers and relevent error logs will be review."
+  - standard_key: NIST-800-53
+# AUDIT REVIEW, ANALYSIS, AND REPORTING | CORRELATE AUDIT REPOSITORIES
+# The organization analyzes and correlates audit records across different repositories to gain organization-wide situational awareness.
+# Supplemental Guidance: Organization-wide situational awareness includes awareness across all three tiers of risk management (i.e., organizational, mission/business process, and information system) and supports cross-organization awareness. Related to: AU-12, IR-4
+    control_key: AU-6 (3)
+    narrative:
+       - key: ""
+         text: "Errors are discussed in slack with relevant parties as they are bing resolved."
```
Comment (ccostino) [inline opencontrol.yaml]: Change `bing` to `being`.

File: opencontrol.yaml [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+    # The organization:
+    # AC-2 (7)(a)
+    # Establishes and administers privileged user accounts in accordance with a role-based access scheme that organizes allowed information system access and privileges into roles;
+    # AC-2 (7)(b)
+    # Monitors privileged role assignments; and
+    # AC-2 (7)(c)
+    # Takes [Assignment: organization-defined actions] when privileged role assignments are no longer appropriate.
+ control_key: AC-2 (7)
+ narrative:
+  - key: "Privileges are set by cloud.gov for most things 18F infrastructure manages the RDS boxes and  FEC and 18F staff control cms privileges."
```
Comment (ccostino) [inline opencontrol.yaml]: Extra space between `and  FEC` toward end and uncapitalized `cms`.

File: opencontrol.yaml [hunk 4]
```diff
@@ -0,0 +1,575 @@
...
+       - key: ""
+         text: ""
+  - standard_key: NIST-800-53
+# IDENTIFICATION AND AUTHENTICATION (ORGANIZATIONAL USERS) | NETWORK ACCESS TO PRIVILEGED ACCOUNTS
+# The information system implements multifactor authentication for network access to privileged accounts.
+# Related to: AC-6
+    control_key: IA-2 (1)
+    narrative:
+       - key: ""
+         text: "Access to applications is set by cloud.gov, 18F infrastructure manages the RDS boxes and FEC and 18F staff control cms access."
```
Comment (ccostino) [inline opencontrol.yaml]: Uncapitalized `cms`.

File: opencontrol.yaml [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+  - standard_key: NIST-800-53
+# IDENTIFICATION AND AUTHENTICATION (ORGANIZATIONAL USERS) | REMOTE ACCESS - SEPARATE DEVICE
+# The information system implements multifactor authentication for remote access to privileged and non-privileged accounts such that one of the factors is provided by a device separate from the system gaining access and the device meets [Assignment: organization-defined strength of mechanism requirements].
+# Supplemental Guidance: For remote access to privileged/non-privileged accounts, the purpose of requiring a device that is separate from the information system gaining access for one of the factors during multifactor authentication is to reduce the likelihood of compromising authentication credentials stored on the system. For example, adversaries deploying malicious code on organizational information systems can potentially compromise such credentials resident on the system and subsequently impersonate authorized users. Related to: AC-6
+    control_key: IA-2 (11)
+    narrative:
+       - key: ""
+         text: "Access to applications is set by cloud.gov, 18F infrastructure manages the RDS boxes and FEC and 18F staff control cms access."
```
Comment (ccostino) [inline opencontrol.yaml]: Uncapitalized `cms`.

File: opencontrol.yaml [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+    narrative:
+       - key: ""
+         text: "We do not have PIV card intergration."
+  - standard_key: NIST-800-53
+# IDENTIFICATION AND AUTHENTICATION (ORGANIZATIONAL USERS) | NETWORK ACCESS TO NON-PRIVILEGED ACCOUNTS
+# The information system implements multifactor authentication for network access to non-privileged accounts.
+    control_key: IA-2 (2)
+    narrative:
+       - key: ""
+         text: "Access to applications is set by cloud.gov, 18F infrastructure manages the RDS boxes and FEC and 18F staff control cms access."
```
Comment (ccostino) [inline opencontrol.yaml]: Uncapitalized `cms`.

File: opencontrol.yaml [hunk 7]
```diff
@@ -0,0 +1,575 @@
...
+# e.  Changing default content of authenticators prior to information system installation;
+# f.  Establishing minimum and maximum lifetime restrictions and reuse conditions for authenticators;
+# g.  Changing/refreshing authenticators [Assignment: organization-defined time period by authenticator type];
+# h.  Protecting authenticator content from unauthorized disclosure and modification;
+# i.  Requiring individuals to take, and having devices implement, specific security safeguards to protect authenticators; and
+# j.  Changing authenticators for group/role accounts when membership to those accounts changes.
+    control_key: IA-5
+    narrative:
+       - key: ""
+         text: "Access to applications is set by cloud.gov, 18F infrastructure manages the RDS boxes and FEC and 18F staff control cms access."
```
Comment (ccostino) [inline opencontrol.yaml]: Uncapitalized `cms`.

File: opencontrol.yaml [hunk 8]
```diff
@@ -0,0 +1,575 @@
...
+# IA-5 (1)(d)
+# Enforces password minimum and maximum lifetime restrictions of [Assignment: organization-defined numbers for lifetime minimum, lifetime maximum];
+# IA-5 (1)(e)
+# Prohibits password reuse for [Assignment: organization-defined number] generations; and
+# IA-5 (1)(f)
+# Allows the use of a temporary password for system logons with an immediate change to a permanent password.
+    control_key: IA-5 (1)
+    narrative:
+       - key: ""
+         text: "Cloud.gov and Amazon handle authentication for the respective systems. For the CMS, Django provides password storage system and uses PBKDF2 by default. (a) Adding a complexity check for cms users (b) Adding a complexity check for cms users (c) CMS Uses Django password storage protections (d) CMS uses Django password storage protections (e) Password reuse is prohibited by cloud.gov and Amazon, not cms (f) Temporary password for system logons with an immediate change to a permanent password on the system level with cloud.gov and Amazon, not cms"
```
Comment (ccostino) [inline opencontrol.yaml]: Several uncapitalized `cms` instances.

File: opencontrol.yaml [hunk 9]
```diff
@@ -0,0 +1,575 @@
...
+       - key: ""
+         text: "Uses GSA, Amazon and GitHub authentication."
+  - standard_key: NIST-800-53
+# AUTHENTICATOR MANAGEMENT | AUTOMATED SUPPORT FOR PASSWORD STRENGTH DETERMINATION
+# The organization employs automated tools to determine if password authenticators are sufficiently strong to satisfy [Assignment: organization-defined requirements].
+# Supplemental Guidance: This control enhancement focuses on the creation of strong passwords and the characteristics of such passwords (e.g., complexity) prior to use, the enforcement of which is carried out by organizational information systems in IA-5 (1). Related to: CA-2, CA-7, RA-5
+    control_key: IA-5 (4)
+    narrative:
+       - key: ""
+         text: "Cloud.gov and Amazon handle authentication for the respective systems. For the CMS, ading a complexity check for cms users."
```
Comment (ccostino) [inline opencontrol.yaml]: Uncapitalized `cms`.

**Comment by LindsayYoung** [comment]
Thank you for your attention to detail! I will get these fixes in after updating the cli and ping you.

**Comment by ccostino** [comment]
I wish there was a :high_five: reaction. :-)  You're welcome!

**Comment by LindsayYoung** [comment]
Most edits are exactly as you suggested. One of the sentences, I just re-did since I thought it was confusing. 

Thanks! Ready for merge @ccostino

**Comment by ccostino** [comment]
Awesome, thank you very much @LindsayYoung!

**Pull request merged**